samples/nametranslation.d.ts.scala

```diff
@@ -0,0 +1,16 @@
+
+import scala.scalajs.js
+import js.annotation._
+import js.|
+
+package nametranslation {
+
+@js.native
+@JSGlobalScope
+object Nametranslation extends js.Object {
+  val uint32Array: js.typedarray.Uint32Array = js.native
+  val readonlyArray: js.Array = js.native
+  val promiseLike: js.Thenable = js.native
+}
+
+}
```

Comment:
Please add an EOL at the end of the file so that git (and GitHub) don't freak out with that ugly symbol.